uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/ArcGISAuthenticationChallengeRelay.cpp

```diff
@@ -0,0 +1,62 @@
+
+/*******************************************************************************
+ *  Copyright 2012-2024 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "ArcGISAuthenticationChallengeRelay.h"
+#include "ArcGISAuthenticationController.h"
+
+// Maps SDK headers
+#include <Authentication/ArcGISAuthenticationChallenge.h>
+#include <Authentication/AuthenticationManager.h>
+#include <ArcGISRuntimeEnvironment.h>
+
+using namespace Esri::ArcGISRuntime;
+using namespace Esri::ArcGISRuntime::Authentication;
+
+namespace Esri::ArcGISRuntime::Toolkit {
+
+/*!
+ \class Toolkit::ArcGISAuthenticationChallengeRelay
+ \inmodule EsriArcGISRuntimeToolkit
+ \ingroup ArcGISQtToolkitUiCppControllers
+ \brief Simple implementation of ArcGISAuthenticationChallengeHandler.
+
+ \internal
```

Comment:
I like this 👍 

I believe this means that this class won't show up in the built doc, correct? Even still, maybe we could add a bit clearer of a brief than this. Maybe something along the lines of "An internal controller to handle ArcGIS challenges for the AuthenticationView"

Reply:
That is correct, it won't appear in the doc at all. I will beef up the explanation.